Xamarin.Forms.Platform.Tizen/Native/DateTimePicker.cs

```diff
@@ -41,19 +41,6 @@ public DateTimePickerMode Mode
 			}
 		}
 
-		public TimeSpan Time
```

Comment:
So things like these are banking change no ?

Reply:
Yes, but above controls that have been modified are being used only internally. So it shouldn’t be a problem.